src/RelationParser.h

```diff
@@ -15,7 +15,7 @@
 namespace snowcrash {
 
     /** Link Relation matching regex */
-    const char* const RelationRegex = "^[[:blank:]]*[Rr]elation[[:blank:]]*:" SYMBOL_IDENTIFIER "?$";
+    const char* const RelationRegex = "^[[:blank:]]*[Rr]elation[[:blank:]]*:[[:blank:]]*([a-z][a-z0-9.-]*)?[[:blank:]]*$";
```

Comment:
No. That piece of regex is specific to link relation identifier as per RFC 5988.